test/service/test_response.py

```diff
@@ -354,7 +348,11 @@ def paginations(self):
                 "count": 2,
                 "order": "desc",
                 "pages": 1,
-                "next": self.base_url + self.path + self.query,
+                "next": self.base_url.set(
+                    path='/index/files',
+                    args=dict(size=5,
+                              search_after='cbb998ce-ddaf-34fa-e163-d14b399c6b34',
+                              search_after_uid='meta%2332')).url,
```

Comment:
aligned indent
```suggestion
                "next": self.base_url.set(path='/index/files',
                                          args=dict(size=5,
                                                    search_after='cbb998ce-ddaf-34fa-e163-d14b399c6b34',
                                                    search_after_uid='meta%2332')).url,
```
alternatively:
```suggestion
                "next": self.base_url.set(
                    path='/index/files',
                    args=dict(size=5,
                              search_after='cbb998ce-ddaf-34fa-e163-d14b399c6b34',
                              search_after_uid='meta%2332')
                ).url,
```
point being the opening and closing parentheses for the call to `set` need to have the same indentation style.